### ISSUE TITLE
Prolonged use of footwear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9964]
**Preferred term label:**
Prolonged use of footwear

**Synonyms**
Wearing closed toe shoes for long periods

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Tinea pedis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Can you provide more detail? Most people wear shoes every day so the utility of this term would depend on there being something unusual?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK470421/
"Prolonged use of tightly fitting footwear"

I added "tightly fitting" to our finding.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001066
- label: Prolonged use of tightly fitting footwear
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:29262247
- def: History of use of tightly fitting shoes for a long period of time.
- comment: This may be a risk factor for tinea pedis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9964
- synonyms:
